wifiphisher/common/dependencies.py

```diff
@@ -0,0 +1,41 @@
+"""Handle dependency check.
+
+This module checks for all the required dependency needed for full
+function.
+"""
+
+from typing import NamedTuple
+from subprocess import (check_call, CalledProcessError)
+from .constants import DN
+
+Result = NamedTuple("Result", [("status", bool), ("name", str)])
```

Comment:
How about a dictionary?

Reply:
not sure why you suggested a dictionary. The current setup will stop on the first missing dependency.

Reply:
What I mean is instead of using a `NamedTuple` here, you may use a `dictionary` (with `tuples` as values) or a class.